### ISSUE TITLE
Search bar not included in logical focus order after toggle button activation

### ISSUE THREAD

**Issue by aviannamiller** [omeka-s-themes/centerrow#72]
After activating the search toggle button via keyboard, tabbing moves focus immediately to the menu nav bar instead of the search bar:
<img width="999" alt="Screenshot of welcome page on Omeka-S site with theme Center Row applied. Focus order, represented by numbered circles connected by lines, shows focus moving from toggle button directly down to menu nav bar" src="https://github.com/user-attachments/assets/e937acd6-ad0a-494c-b298-a3cd92b3bdd1" />
The search bar and button are indeed focusable elements, but, as it stands, this not a logical order. Focus should move into the text field, then to the search button, followed by either returning to the original trigger element, the search toggle button, (optional) or moving to the nav. A model of how this might looks is as follows:
<img width="996" alt="Screenshot of welcome page. Focus order, represented by numbered circles connected by lines, shows focus moving from toggle button to search text input to search submit button to toggle button, then down to menu nav bar" src="https://github.com/user-attachments/assets/1f5cbd5d-afe6-45b6-af64-efc87654a1aa" />

**Comment by kimisgold** [comment]
This used to be in effect, but js needed to update to match new input text id. Should be fixed now.

**Comment by aviannamiller** [comment]
Thanks! I tested across screen readers, and this looks to be fixed.
